web/thesauruses/kotlin/1.5/operators.json

```diff
@@ -0,0 +1,226 @@
+      },
+      "integer_division": {
+        "name": "Integer division operator",
+        "code": "a // b"
+      },
+      "integer_division_assignment": {
+        "name": "Integer division and assignment operator",
+        "code": "a //= b"
+      },
+      "modulus": {
+        "name": "Modulus (remainder) operator",
+        "code": "a % b"
+      },
+      "modulus_assignment": {
+        "name": "Modulus and assignment operator",
+        "code": "a %= b"
+      },
+      "unary_plus": {
+        "name": "Unary plus operator",
+        "code": "+a"
+      },
+      "unary_minus": {
+        "name": "Unary minus operator",
+        "code": "-a"
+      },
+      "increment": {
+        "name": "Increment (add 1) operator",
+        "code": [
+            "a++ // postfix",
+            "++a // prefix"
+        ],
+        "comment": "The prefix form returns the new value, the postfix form returns the old value."
+      },
+      "decrement": {
+        "name": "Decrement (subtract 1) operator",
+        "code": [
+            "a-- // postfix",
+            "--a // prefix"
+        ],
+        "comment": "The prefix form returns the new value, the postfix form returns the old value."
+      },
+      "exponential": {
+        "name": "Exponential operator",
+        "code": "Math.pow(a, b)",
+        "comment": "Not a operator but a standard library function."
+      },
+      "factorial": {
+        "not-implemented": true,
+        "name": "Factorial operator"
+      },
+      "absolute_value": {
+        "name": "Absolute value operator",
+        "code": "Math.abs(a)"
+      },
+      "percentage": {
+        "not-implemented": true,
+        "name": "Percentage operator"
+      },
+      "equal_to": {
+        "name": "Equality operator",
+        "code": [
+          "a == b // structural equality",
+          "a === b // referential equality"
+        ],
+        "comment": "referential equality `===` checks if two references point to the same object, while structural equality `==` checks if two objects have the same content."
+      },
+      "not_equal_to": {
+        "name": "Not equal to operator",
+        "code": [
+            "a != b // structural inequality",
+            "a !== b // referential inequality"
+        ],
+        "comment": "referential equality `!==` checks if two references fo not point to the same object, while structural equality `!=` checks if two objects do not have the same content."
+      },
+      "less_than": {
+        "name": "Less than operator",
+        "code": "a < b"
+      },
+      "less_than_or_equal_to": {
+        "name": "Less than or equal to operator",
+        "code": "a <= b"
+      },
+      "greater_than": {
+        "name": "Greater than operator",
+        "code": "a > b"
+      },
+      "greater_than_or_equal_to": {
+        "name": "Greater than or equal to operator",
+        "code": "a >= b"
+      },
+      "null_coalescing": {
+        "name": "Elvis operator",
```

Comment:
The name doesn't technically show up, it's pulled in from the meta file. It'll still show up as a null-coalescing operator. You might want to clarify in the comment that's the case here (and also Kotlin calls it that)